diff --git a/src/components/Footer.vue b/src/components/Footer.vue
index f941a2e..b25e409 100644
--- a/src/components/Footer.vue
+++ b/src/components/Footer.vue
@@ -1,7 +1,10 @@
 <template>
   <v-footer app absolute padless>
-    <v-col class="text-center" cols="12">
+    <v-col class="text-center" cols="12" color="grey">
       {{ new Date().getFullYear() }} — <strong>{{ title }}</strong>
+      <v-btn icon color="grey" href="https://github.com/joaxcar/bookbook">
+        <v-icon>mdi-github-circle</v-icon>
+      </v-btn>
     </v-col>
   </v-footer>
 </template>
@@ -16,3 +19,10 @@ export default {
   }
 };
 </script>
+
+<style scoped>
+text-center {
+  color: grey;
+  font-weight: bold;
+}
+</style>
diff --git a/src/data/GoogleAPI.js b/src/data/GoogleAPI.js
index 1422247..11372be 100644
--- a/src/data/GoogleAPI.js
+++ b/src/data/GoogleAPI.js
@@ -1,5 +1,4 @@
 let BASE_URL = "https://www.googleapis.com/books/v1/volumes?";
-let BASE_URL_ID = "https://www.googleapis.com/books/v1/volumes";
 
 function processResponse(response) {
   if (response.ok) {
@@ -51,10 +50,4 @@ function getBook(isbn) {
     });
 }
 
-export function getBookById(id) {
-  return fetch(`${BASE_URL_ID}/${id}`)
-    .then(processResponse)
-    .then(data => makeData(data));
-}
-
 export default getBook;
diff --git a/src/data/fs.js b/src/data/fs.js
index a0d34eb..7916fe8 100644
--- a/src/data/fs.js
+++ b/src/data/fs.js
@@ -24,7 +24,7 @@ const fs = {
     db.collection("users")
       .doc(user.uid)
       .set({ displayName: user.displayName, email: user.email })
-      .then(() => window.console.log("user added successfully"))
+      .then(() => window.console.log("user added: " + user))
       .catch(err => window.console.error("firestore adduser error: " + err));
   },
   subscribe: uid => {
diff --git a/src/router/index.js b/src/router/index.js
index 09b705d..efec2c8 100644
--- a/src/router/index.js
+++ b/src/router/index.js
@@ -30,6 +30,11 @@ const routes = [
     name: "Dashboard",
     component: Dashboard
   },
+  {
+    path: "/dashboard/:type",
+    name: "DashType",
+    component: Dashboard
+  },
   {
     path: "/about",
     name: "about",
@@ -40,7 +45,7 @@ const routes = [
       import(/* webpackChunkName: "about" */ "../views/About.vue")
   },
   {
-    path: "/details/:id",
+    path: "/details",
     name: "Details",
     component: BookDetails
   },
diff --git a/src/store/index.js b/src/store/index.js
index 3a5c51f..1bd343b 100644
--- a/src/store/index.js
+++ b/src/store/index.js
@@ -21,9 +21,6 @@ export default new Vuex.Store({
   getters: {
     user(state) {
       return state.user;
-    },
-    book: state => id => {
-      return state.data.books.filter(book => book.id == id);
     }
   },
   mutations: {
diff --git a/src/views/BookDetails.vue b/src/views/BookDetails.vue
index f072721..6afbd95 100644
--- a/src/views/BookDetails.vue
+++ b/src/views/BookDetails.vue
@@ -1,75 +1,7 @@
 <template>
-  <v-container>
-    <div v-if="!loading">
-      <v-row align="center" justify="center">
-        <v-col cols="12" md="8">
-          <v-card tile>
-            <div class="d-flex flex-no-wrap">
-              <div>
-                <v-img width="60px" :src="book.imageLinks.thumbnail"> </v-img>
-              </div>
-              <div>
-                <v-card-text>
-                  <div>
-                    {{ book.title }}
-                  </div>
-                  <div>{{ book.publisher }}</div>
-                  <div>{{ book.authors[0] }}</div>
-                </v-card-text>
-              </div>
-              <v-spacer />
-            </div>
-            <div>
-              <v-card-actions>
-                <v-btn v-if="true" text color="purple">
-                  Add to library
-                </v-btn>
-                <v-btn v-else text color="green">In My Books ✔</v-btn>
-
-                <v-spacer></v-spacer>
-
-                <v-btn icon @click="book.show = !book.show">
-                  <v-icon>{{
-                    book.show ? "mdi-chevron-up" : "mdi-chevron-down"
-                  }}</v-icon>
-                </v-btn>
-              </v-card-actions>
-            </div>
-
-            <div v-if="book.show">{{ book.description }}</div>
-          </v-card>
-        </v-col>
-      </v-row>
-    </div>
-  </v-container>
+  <h1>Book Details view</h1>
 </template>
 
-<script>
-import getBookById from "@/data/GoogleAPI";
-export default {
-  data() {
-    return {
-      book: {},
-      loading: false
-    };
-  },
-  activated() {
-    this.getBookInfo();
-  },
-  methods: {
-    getBookInfo: function() {
-      this.loading = true;
-      const id = this.$route.params.id;
-      getBookById(id).then(ret => (this.book = ret.items[0]));
-      this.loading = false;
-    }
-  },
-  computed: {
-    // book() {
-    //   return this.$store.getters.book(this.$route.params.id);
-    // }
-  }
-};
-</script>
+<script></script>
 
 <style scoped></style>
diff --git a/src/views/Dashboard.vue b/src/views/Dashboard.vue
index 317fa5a..6f140dd 100644
--- a/src/views/Dashboard.vue
+++ b/src/views/Dashboard.vue
@@ -3,7 +3,7 @@
     <v-row dense align="center" justify="center">
       <v-col cols="12" md="8">
         <v-text-field
-          label="Search (pres enter with prefilled content for example)"
+          label="Search for isbn/title/author/publisher"
           filled
           rounded
           v-model="searchText"
@@ -16,6 +16,7 @@
           @keydown.enter="getBook"
           @click:append-outer="getBook"
           @click:append="toggleScan"
+          ref="textArea"
         ></v-text-field>
       </v-col>
     </v-row>
@@ -29,30 +30,32 @@
         <v-card v-for="item in books" :key="item.id" tile>
           <div class="d-flex flex-no-wrap">
             <div>
-              <router-link :to="{ name: 'Details', params: { id: item.id } }">
-                <v-img width="60px" :src="item.imageLinks.thumbnail"> </v-img>
-              </router-link>
+              <v-img width="60px" :src="item.imageLinks.thumbnail"></v-img>
             </div>
             <div>
               <v-card-text>
-                <div>
-                  <router-link
-                    :to="{ name: 'Details', params: { id: item.id } }"
-                    >{{ item.title }}</router-link
-                  >
+                <div class="body-2 font-weight-medium">{{ item.title }}</div>
+                <span
+                  v-for="author in item.authors"
+                  :key="item.id + author"
+                  class="caption"
+                >
+                  {{ author }}
+                </span>
+                <div class="caption font-weight-light">
+                  {{ item.publisher }}
                 </div>
-
-                <div>{{ item.publisher }}</div>
               </v-card-text>
             </div>
             <v-spacer />
-            <div>
+            <div justify="center">
               <v-card-actions>
                 <!-- todo CHANGE BTN WHEN BOOK ALREADY IN LIBRARY  -->
                 <v-btn
                   v-if="inLib(item)"
                   @click="() => addBook(item)"
                   outlined
+                  small
                   color="purple"
                 >
                   Add to library
@@ -82,7 +85,7 @@ export default {
     return {
       message: "",
       books: [],
-      searchText: "9781405924412",
+      searchText: "",
       library: [],
       show: false,
       isScanning: false,
@@ -122,6 +125,19 @@ export default {
       }.bind(this),
       2000
     );
+    this.$nextTick(() => this.$refs.textArea.focus());
+  },
+  activated() {
+    this.$nextTick(() => this.$refs.textArea.focus());
+    window.console.log(this.$route.params.type);
+    if (this.$route.params.type === "camera") {
+      this.toggleScan();
+    }
+  },
+  deactivated() {
+    if (this.isScanning) {
+      this.toggleScan();
+    }
   },
   methods: {
     get: getsBook,
diff --git a/src/views/MyBooks.vue b/src/views/MyBooks.vue
index c1d35d0..76fa1e6 100644
--- a/src/views/MyBooks.vue
+++ b/src/views/MyBooks.vue
@@ -14,6 +14,7 @@
           @input="debounceFilterBooks"
           @keydown.enter="filterBooks"
           @click:append-outer="filterBooks"
+          ref="textArea"
         ></v-text-field>
       </v-col>
     </v-row>
@@ -47,9 +48,17 @@
           </div>
           <div>
             <v-card-text>
-              <div>{{ item.title }}</div>
-              <div>{{ item.authors[0] }}</div>
-              <div>{{ item.publisher }}</div>
+              <div class="body-2 font-weight-medium">{{ item.title }}</div>
+              <span
+                v-for="author in item.authors"
+                :key="item.id + author"
+                class="caption"
+              >
+                {{ author }}
+              </span>
+              <div class="caption font-weight-light">
+                {{ item.publisher }}
+              </div>
             </v-card-text>
           </div>
           <v-spacer />
@@ -69,33 +78,24 @@
     </v-row>
     <v-speed-dial
       v-model="fab"
+      bottom
+      right
+      absolute
+      fixed
       direction="top"
-      top
-      transition="slide-y-reverse-transition"
+      transition="slide-x-transition"
     >
       <template v-slot:activator>
-        <v-btn
-          color="success"
-          v-model="fab"
-          fab
-          x-large
-          dark
-          fixed
-          right
-          bottom
-        >
+        <v-btn color="success" v-model="fab" fab large>
           <v-icon v-if="fab">mdi-close</v-icon>
           <v-icon v-else>mdi-plus</v-icon>
         </v-btn>
       </template>
-      <v-btn fab dark small color="green">
-        <v-icon>mdi-pencil</v-icon>
-      </v-btn>
-      <v-btn fab dark small color="indigo">
-        <v-icon>mdi-plus</v-icon>
+      <v-btn fab dark small color="green" to="/dashboard">
+        <v-icon>mdi-format-text</v-icon>
       </v-btn>
-      <v-btn fab dark small color="red">
-        <v-icon>mdi-delete</v-icon>
+      <v-btn fab dark small color="indigo" to="/dashboard/camera">
+        <v-icon>mdi-camera</v-icon>
       </v-btn>
     </v-speed-dial>
   </v-container>
@@ -131,6 +131,10 @@ export default {
       400
     );
     this.books = [...this.data.books];
+    this.$nextTick(() => this.$refs.textArea.focus());
+  },
+  activated() {
+    this.$nextTick(() => this.$refs.textArea.focus());
   },
   methods: {
     debounce: Debounce,
diff --git a/src/views/Welcome.vue b/src/views/Welcome.vue
index 2cac0f5..ff5bb02 100644
--- a/src/views/Welcome.vue
+++ b/src/views/Welcome.vue
@@ -31,14 +31,6 @@
     <section>
       <v-row>
         <v-col align="center" justify="center" cols="12" light>
-          <v-row>
-            <v-col class="pt-7">
-              <h2 class="font-weight-bold">
-                Find and store all your books
-              </h2>
-              Search for ISBN or by barcode scanner
-            </v-col>
-          </v-row>
           <v-row>
             <v-col>
               <v-card
@@ -64,40 +56,6 @@
         </v-col>
       </v-row>
     </section>
-
-    <section>
-      <v-row>
-        <v-col justify="center" align="center">
-          <v-card class="elevation-0 transparent">
-            <v-card-title primary-title class="layout justify-center">
-              <div class="headline">Group info</div>
-            </v-card-title>
-            <v-card-text>
-              We are a group of students doing this for the first time. Dont
-              expect greatnes!
-            </v-card-text>
-          </v-card>
-          <v-card class="elevation-0 transparent">
-            <v-card-title primary-title class="layout justify-center">
-              <div class="headline">Project code</div>
-            </v-card-title>
-            <v-card-text>
-              Project source code can be found at
-              <a href="https://github.com/joaxcar/bookbook">GitHub</a>
-            </v-card-text>
-          </v-card>
-        </v-col>
-      </v-row>
-    </section>
-
-    <v-row>
-      <v-col align="center" class="display-1">Example API-data: </v-col>
-    </v-row>
-    <v-row>
-      <v-col v-if="book.items" align="center" cols="12" class="testAPI">{{
-        book.items[0].volumeInfo
-      }}</v-col>
-    </v-row>
   </v-container>
 </template>
 
@@ -112,18 +70,18 @@ export default {
       infobox: [
         {
           title: "Search for books",
-          icon: "mdi-book",
-          text: `This text explains how a search is made. How it is awsome, and how to move forward!`
+          icon: "mdi-barcode-scan",
+          text: `Click the camera in the search bar and you're a tap away from adding your first book. It's that easy!`
         },
         {
-          title: "Store your thoughts",
-          icon: "mdi-book-open-page-variant",
-          text: `This text explains how the data is stored. How it is awsome, and how to move forward!`
+          title: "Organize",
+          icon: "mdi-bookshelf",
+          text: `Create different shelves! With an ON LOAN shelf, you'll never forget who borrowed that first edition poetry collection that you love.`
         },
         {
-          title: "Organize",
-          icon: "mdi-book-open",
-          text: `This text explains how to organize books. How it is awsome, and how to move forward!`
+          title: "Review",
+          icon: "mdi-star-half",
+          text: `What did you think about the book you just finished? Rate it and review it for future reference.`
         }
       ],
       book: {}
